packages/slice-machine/components/ScreenshotChangesModal/Views/index.tsx

```diff
@@ -58,16 +72,49 @@ function VariationScreenshot({
     onHandleFile: generateSliceCustomScreenshot,
   });
 
+  const sliceView = useMemo(
+    () =>
+      slice
+        ? [
+            {
+              sliceID: slice.model.id,
+              variationID: variationID,
+            },
+          ]
+        : null,
+    [slice.model.id, variationID]
+  );
+
+  if (!sliceView) return <></>;
+
+  const openSimulator = () =>
+    checkSimulatorSetup(true, () =>
+      window.open(
```

Comment:
OPT: Mathieu suggested a small improvement by giving is a second parameter which would be the ID, that way we can avoid to re-open pages for the same slice / variationID.